src/PlaneWaveBasis.jl

```diff
@@ -245,13 +246,58 @@ build_kpoints(basis::PlaneWaveBasis, kcoords) =
     basis
 end
 
+# This is the "internal" constructor; the higher-level one below should be preferred
+@timing function PlaneWaveBasis(model::Model{T}, Ecut::Number,
+                                kcoords::AbstractVector, ksymops, symmetries=nothing;
+                                fft_size=nothing, variational=true,
+                                optimize_fft_size=false, supersampling=2,
+                                kgrid=nothing, kshift=nothing,
+                                comm_kpts=MPI.COMM_WORLD,
+                               ) where {T <: Real}
+    mpi_ensure_initialized()
```

Comment:
Why is this needed?

Reply:
because we call MPI and it might be the first time we do

Reply:
but the mpi call happens in the internal constructor, right? so there is another init call

Reply:
ah sorry I was not at the right line. Yup you're right